### PR TITLE
Add sections 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- The website has no section but the hero shot. #25 #26 
- The hero component does not fit the theme. #27 

**What is the new behavior (if this is a feature change)?**
- The website now has two additional sections. #25 #26 
- The hero component has been improved to fit the theme. #27

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
After a line-length change it should be okay.

File: package.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
     "react-dom": "^16.8.6",
     "react-fade-in": "^0.1.6",
     "react-html-parser": "^2.0.2",
+    "react-loadable": "^5.5.0",
+    "react-loadable-visibility": "^3.0.1",
+    "react-particles-js": "^2.7.0",
```
Comment (kleberbaum) [inline package.json]: ParticlesJS should be removed from package.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: package.json (hunk 1 above)
Comment (Aichnerc) [reply package.json]: This problem has been resolved.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by kleberbaum — APPROVED** [review]

**Pull request merged**